packages/core/admin/_internal/node/develop.ts

```diff
@@ -104,24 +148,44 @@ const develop = async ({
   }
 
   if (cluster.isWorker) {
+    const strapi = strapiFactory({
```

Comment:
Just a random idea, but we could also add a loader for the Strapi instance creation & loading (if it can take more than 1 or 2 seconds)
It would only be helpful for debugging app crashes but hey :shrug: